assets/components/inlinePaymentLogos/inlinePaymentLogos.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import Svg from 'components/svg/svg';
+
+// ----- Component ----- //
+
+export default function InlinePaymentLogos() {
+
+  return (
+    <div className={'component-inline-payment-logos'}>
+      <Svg svgName="visa-logo" />
+      <Svg svgName="mastercard-logo" />
+      <Svg svgName="paypal-logo" />
```

Comment:
Good catch @davidfurey so far we want to turn on PayPal in all the channels, after this afternoon test with apps. We can see if we decide to turn off PayPal for that scenario. In that case, a following PR will be created.